docs/topics/proc_creating-nodejs-app.adoc

```diff
@@ -65,6 +65,7 @@ $ curl http://localhost:8080
 {"content":"Hello, World!"}
 ----
 
+[#deploy-existing-nodejs-app-openshift]
```

Comment:
I'm not sure it's the best idea to call this section `deploy-existing-nodejs-app-openshift`, when you link to it from some other section that's also called "proc_deploy-existing-nodejs-app.adoc".